docs/src/docs/common-pitfalls.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Common pitfalls
+---
+
+For the most part, you should be able to seamlessly migrate your site from 11ty to Slinkity with minimal reconfiguration. But there are a few changes that may catch you off guard. These gotchas are summarized below.
+
+## Certain files disappeared from my production build output folder
+
+In an 11ty site, you are free for the most part to structure your project however you like. You can put your static assets in a `public` folder outside your [11ty input directory]((https://www.11ty.dev/docs/config/#input-directory)) and pass-through copy them, but you don't need to—you could also store all of your assets inside your 11ty input directory. It doesn't matter which approach you take, so long as you configure your 11ty project accordingly and write your files to the correct locations. However, you need to take a slightly different approach in Slinkity due to the standards enforced by Vite.
+
+First, we need to understand that a Slinkity build consists of multiple steps:
+
+1. Processing your site with 11ty and dumping the result into a temporary build folder (e.g., `.11ty-build-DcNVBN`).
+2. Processing the temporary build folder with Vite to bundle assets on the pages that reference them.
+3. Writing the temporary build folder to your 11ty output directory (e.g., `_site`)
+4. Deleting the temporary build folder.
```

Comment:
That was mainly needed because of the phrasing before the list: `First, we need to understand that a Slinkity build consists of multiple steps:` (i.e., "A slinkity build consists of verbing this, verbing that, etc."). We can change it to be:

```
First, we need to understand how Slinkity builds work. In a typical build, we:
```

And then this suggestion works 👍